web/src/app/components/social-cards/social-cards.component.html

```diff
@@ -0,0 +1,93 @@
+<mat-card class="outer-container" fxLayout="row" fxLayoutGap="20px">
+  <mat-card fxFlex="25">
+    <mat-card-content *ngIf="userSocial.github != undefined else github">
+      <mat-icon svgIcon="github" class="github"></mat-icon>
+      <div fxLayout class="stats-wrapper">
+        <div fxLayout="column" fxFlex="33">
+          <span class="stats-display">{{userSocial.github.followers}}</span>
```

Comment:
I think it is recommended (and what we have been using?) to have spaces around the variables

`{{a}}` to `{{ a }}`

Reply:
Done @eddiejaoude . The formatter didn't add the spaces. Fixed it now